vnext/ReactUWP/Modules/NativeUIManager.cpp

```diff
@@ -93,6 +93,11 @@ void NativeUIManager::DirtyYogaNode(int64_t tag)
   }
 }
 
+void NativeUIManager::AddBatchCompletedCallback(std::function<void()> callback)
```

Comment:
try to move std::functions rather than copying, take rvalue ref, move into callback vector

Reply:
You're moving into the m_batchCompletedCallbacks, are we moving in the call to AddBatchCompletedCallback too?